app_flutter/lib/state/flutter_build.dart

```diff
@@ -26,6 +26,9 @@ class FlutterBuildState extends ChangeNotifier {
   /// The current status of the commits loaded.
   List<CommitStatus> statuses = [];
 
+  /// Whether or not flutter/flutter currently passes tests.
+  bool isTreeBuilding = true;
```

Comment:
This should probably be a getter for a private field. That will prevent it from being accidentally set by consuming code:

```
bool get isTreeBuilding => _isTreeBuilding;
bool _isTreeBuilding = true;
```

Reply:
Good point, added